wagtail/core/models.py

```diff
@@ -2760,13 +2764,34 @@ def get_view_restrictions(self):
         """Return a query set of all collection view restrictions that apply to this collection"""
         return CollectionViewRestriction.objects.filter(collection__in=self.get_ancestors(inclusive=True))
 
-    @staticmethod
-    def order_for_display(queryset):
-        return queryset.annotate(
-            display_order=Case(
-                When(depth=1, then=Value('')),
-                default='name')
-        ).order_by('display_order')
+    def get_indented_name(self, indentation_start_depth=2, plain_text=False):
```

Comment:
What is "default argument value pollution"? I've heard of using `None` as a default when you would have otherwise used a mutable value, like an empty list, since if you change the parameter value within the function, the default value changes for the next call. But I've never heard of that being an issue with scalars like `int`.

Reply:
Okay, my bad. I was making referring to https://florimond.dev/blog/articles/2018/08/python-mutable-defaults-are-the-source-of-all-evil/, but I can see now that `int` values are immutable. I retract this request :)